pkg/sdk/helper_test.go

```diff
@@ -320,12 +330,16 @@ func createWarehouseWithOptions(t *testing.T, client *Client, opts *CreateWareho
 
 func createDatabase(t *testing.T, client *Client) (*Database, func()) {
 	t.Helper()
-	return createDatabaseWithOptions(t, client, &CreateDatabaseOptions{})
+	return createDatabaseWithOptions(t, client, randomAccountObjectIdentifier(t), &CreateDatabaseOptions{})
 }
 
-func createDatabaseWithOptions(t *testing.T, client *Client, _ *CreateDatabaseOptions) (*Database, func()) {
+func createDatabaseWithIdentifier(t *testing.T, client *Client, id AccountObjectIdentifier) (*Database, func()) {
+	t.Helper()
+	return createDatabaseWithOptions(t, client, id, &CreateDatabaseOptions{})
+}
+
+func createDatabaseWithOptions(t *testing.T, client *Client, id AccountObjectIdentifier, _ *CreateDatabaseOptions) (*Database, func()) {
```

Comment:
I saw that many functions are '...WithOptions', but they're discarded (_ value) at parameter level. Is there a point in making such arg ? If not maybe we could stop adding such args and remove the other ones later on

Reply:
Done.